sql/core/src/test/scala/org/apache/spark/sql/execution/command/ShowPartitionsSuiteBase.scala

```diff
@@ -149,4 +149,28 @@ trait ShowPartitionsSuiteBase extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("case sensitivity of partition spec") {
```

Comment:
Shall we add a JIRA prefix @MaxGekk?

Reply:
The test already has prefixes V1, V2 or Hive V1. If I add one more prefix, this will look not beauty, though...